elixir/apps/domain/lib/domain/gateways.ex

```diff
@@ -419,30 +411,6 @@ defmodule Domain.Gateways do
     end
   end
 
-  def connect_gateway(%Gateway{} = gateway) do
-    with {:ok, _} <-
-           Presence.track(
-             self(),
-             group_gateways_presence_topic(gateway.group_id),
-             gateway.id,
-             %{}
-           ),
-         {:ok, _} <-
-           Presence.track(
-             self(),
-             account_gateways_presence_topic(gateway.account_id),
-             gateway.id,
-             %{
-               online_at: System.system_time(:second)
-             }
-           ) do
-      :ok = PubSub.subscribe(gateway_topic(gateway))
-      :ok = PubSub.subscribe(group_gateways_topic(gateway.group_id))
-      :ok = PubSub.subscribe(account_gateways_topic(gateway.account_id))
```

Comment:
The `account_gateways_topic` wasn't being used anywhere. The presence, yes (for UI), but not the underlying topic itself. The gateways in an account are disconnected when their token is deleted, similar to group.